src/ostorlab/runtimes/local/app/common.py

```diff
@@ -0,0 +1,44 @@
+"""common utilities for the flask app."""
+
+from typing import Optional
+
+import graphene
+import cvss
+
+
+class SortEnum(graphene.Enum):
+    """Sort enum, for the sorting order of the results."""
+
+    ASC = 1
+    DESC = 2
+
+
+class RiskRatingEnum(graphene.Enum):
```

Comment:
Is this new, we should already have this in the model.

Reply:
this is graphene.Enum, the one we have is enum.Enum.

Reply:
and graphene.enum.from_enum didnt work